src/TraceEvent/Computers/TraceManagedProcess.cs

```diff
@@ -121,6 +121,11 @@ public class TraceLoadedDotNetRuntime
         /// Returns the .NET startup flags
         /// </summary>
         public StartupFlags StartupFlags { get; internal set; }
+        /// <summary>
+        /// Date and time of when the runtime was built
+        /// This is useful when a more detailed version is not present
+        /// </summary>
+        public DateTime RuntimeBuiltTime { get; internal set; }
```

Comment:
Thanks for the explanation. Due to the number of times I've seen these two items cause actual problems, I'm still going to leave a suggestion. With that said, I understand that this could pretty easily be a "I wish it could be this way" type of situation where the current form ends up being what we're stuck with for some reason or another.

:bulb: Consider using `DateTimeOffset?` for this property as an API guard against certain possible confusion about its meaning/use throughout the portions of the code where it appears.